config/config.go

```diff
@@ -193,6 +193,7 @@ type GlobalConfig struct {
 	SlackURL        string `yaml:"slack_url"`
 	PagerdutyURL    string `yaml:"pagerduty_url"`
 	OpsGenieAPIHost string `yaml:"opsgenie_api_host"`
+	ExternalURL     string `yaml:"external_url"`
```

Comment:
Me neither, still find it preferable. But not feeling strongly right now if it unblocks the rest.